### PR TITLE
Added cheatsheet for Japanese Language

### PR DESCRIPTION
**Pull request by MathewtheCoder**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Japanese Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: 
Cheatsheet for japanese language showing meaning of all common words and phrases.

---
https://duck.co/ia/view/japanese_language_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/japanese.json [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            "key":"3",
+            "val":"san"
+         },
+         {
+            "key":"4",
+            "val":"yon"
+         },
+         {
+            "key":"go",
+            "val":"runs a script as a background thread"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/japanese.json]: is this a typo?

Comment (MathewtheCoder) [inline share/goodie/cheat_sheets/json/japanese.json]: Yeah it is thanks for pointing it out i corrected it.

File: share/goodie/cheat_sheets/json/japanese.json [hunk 2]
```diff
@@ -0,0 +1,283 @@
+{
+   "id":"japanese_cheat_sheet",
+   "name":"Japanese Cheatsheet",
+   "description":"Cheatsheet showing meanings of commonly used japanese words & phrases",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/japanese.json]: We shouldn't have `Cheatsheet` in the name or the description. Maybe `Basic Japanese` and Common Japanese words and phrases` respectively?

File: share/goodie/cheat_sheets/json/japanese.json [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+            "key":"2",
+            "val":"ni"
+         },
+         {
+            "key":"3",
+            "val":"san"
+         },
+         {
+            "key":"4",
+            "val":"yon"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/japanese.json]: Also `shi` is used for 4

File: share/goodie/cheat_sheets/json/japanese.json [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+            "val":"Thank you very much"
+         },
+         {
+            "key":"itadakimasu",
+            "val":"Thanks for the food"
+         }
+      ],
+      "Greetings":[
+         {
+            "key":"Mathew desu",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/japanese.json]: More traditionally spelled `Matthew"

File: share/goodie/cheat_sheets/json/japanese.json (hunk 2 above)
Comment (MathewtheCoder) [inline share/goodie/cheat_sheets/json/japanese.json]: Yeah i corrected it

File: share/goodie/cheat_sheets/json/japanese.json [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+         {
+            "key":"oyasuminasai",
+            "val":"Good Night"
+         },
+         {
+            "key":"sayonara",
+            "val":"GoodBye"
+         },
+         {
+            "key":"Ja Mata",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/japanese.json]: for consistency `ja mata`

**Comment by mintsoft** [comment]
@tagawa Seems appropriate to assign this to you

File: share/goodie/cheat_sheets/json/japanese.json (hunk 3 above)
Comment (MathewtheCoder) [inline share/goodie/cheat_sheets/json/japanese.json]: I searched the net and found out that japanese people tend to use yon to refer to 4 instead of shi

Comment (MathewtheCoder) [inline share/goodie/cheat_sheets/json/japanese.json]: Nevertheless i have added shi to 4 and will wait for @tagawa opinion

**Comment by edgesince84** [comment]
@mintsoft I think you're right, @tagawa should be able to help us here 😀 

thanks @MathewtheCoder

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 6]
```diff
@@ -0,0 +1,283 @@
+{
+   "id":"japanese_language_cheat_sheet",
+   "name":"Basic Japanese",
+   "description":"A guide to basic japanese",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: `japanese` should be `Japanese`

Comment (MathewtheCoder) [inline share/goodie/cheat_sheets/json/japanese-language.json]: I made the change you requested

**Comment by tagawa** [comment]
Thanks for the quick changes @MathewtheCoder. We actually already have a submission for a Japanese cheat sheet: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2635 but it's been abandoned. It does have some good feedback and includes Japanese characters, however. I'm now checking about what the best way to proceed is.

**Comment by MathewtheCoder** [comment]
I am trying to use language template should i upload that or just stay with this one

**Comment by tagawa** [comment]
@MathewtheCoder I'd say wait for a while until we decide how we should deal with the two cheat sheets. I'd like to make sure there's the minimum extra work for you or the other developer.

**Comment by MathewtheCoder** [comment]
@tagawa Ok Thanks

**Comment by mintsoft** [comment]
@tagawa It'd be nice to also have the hiragana in addition to the romajii like that one one did

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Japanese Language cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/japanese_language_cheat_sheet_mobile.png?hECLl8LuQT)

**Comment by tagawa** [comment]
@MathewtheCoder Sorry for the delay. Would  you be able to incorporate some of the terms and feedback from the [previous cheat sheet](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2635) into your cheat sheet, including the Japanese characters? I can help with some of that if you get stuck.

**Comment by MathewtheCoder** [comment]
@tagawa Yeah i will do that but i kind of need some help with the japnese words

**Comment by tagawa** [comment]
@MathewtheCoder Thanks. If you want, you could just do the "English alphabet" text first, i.e. ignore the Japanese characters, and I can help add those later.

**Comment by MathewtheCoder** [comment]
@tagawa Yes i will try to do the japanese part as much as possible thanks for your help

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 7]
```diff
@@ -21,262 +21,329 @@
...
+            "trn":"san"
          },
          {
             "key":"4",
-            "val":"shi/yon"
+            "val":"四",
+            "trn":"shi/yon"
          },
          {
             "key":"go",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: The key should be `5`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 8]
```diff
@@ -21,262 +21,329 @@
...
-            "val":"5"
+            "val":"五",
+            "trn":"5"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: trn should be `go`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 9]
```diff
@@ -21,262 +21,329 @@
...
          },
          {
             "key":"roku",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: key should be `6`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 10]
```diff
@@ -21,262 +21,329 @@
...
-            "val":"6"
+            "val":"六",
+            "trn":"6"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: trn should be `roku`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 11]
```diff
@@ -21,262 +21,329 @@
...
          },
          {
             "key":"nana",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: key should be `7`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 12]
```diff
@@ -21,262 +21,329 @@
...
-            "val":"7"
+            "val":"七",
+            "trn":"7"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: trn should be `nana/shichi`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 13]
```diff
@@ -21,262 +21,329 @@
...
          {
             "key":"itsu",
-            "val":"When"
+            "val":"いつ",
+            "trn":"When"
          },
          {
             "key":"nan desu ka",
-            "val":"What"
+            "val":"ナンデスか",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: val should be `何ですか`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 14]
```diff
@@ -21,262 +21,329 @@
...
-            "val":"See you Tomorrow"
+            "val":"マタ明日",
+            "trn":"See you Tomorrow"
          }
       ],
       "Questions":[
          {
             "key":"dare desu ka",
-            "val":"Who is it"
+            "val":"あえてデスか",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: val should be `誰ですか`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 15]
```diff
@@ -21,262 +21,329 @@
...
-            "val":"Thanks for the food"
+            "val":"いただきます。",
+            "trn":"Thanks for the food"
          }
       ],
       "Greetings":[
          {
             "key":"Matthew desu",
-            "val":"I'm Mathew"
+            "val":"マシューのデス",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: val should be `マシューです`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 16]
```diff
@@ -21,262 +21,329 @@
...
-            "key":"o-namae-wa",
-            "val":"What's your name"
+            "key":"Anata no namae wa nandesuka",
+            "val":"あなたの名前は何ですか",
+            "trn":"What's your name"
          },
          {
             "key":"yoroshiku onegai shimasu",
-            "val":"I'm pleased to meet you"
+            "val":"ヨロシクおねがい",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: val should be `よろしくお願いします`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 17]
```diff
@@ -21,262 +21,329 @@
...
          {
             "key":"kon-ban-wa",
-            "val":"Good Afternoon"
+            "val":"こんばんは",
+            "trn":"Good Afternoon"
          },
          {
             "key":"oyasuminasai",
-            "val":"Good Night"
+            "val":"おやすみ",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: val should be `おやすみなさい`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 18]
```diff
@@ -21,262 +21,329 @@
...
+            "trn":"Good Night"
          },
          {
             "key":"sayonara",
-            "val":"GoodBye"
+            "val":"サヨナラ",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: val should be `さようなら`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 19]
```diff
@@ -21,262 +21,329 @@
...
          {
             "key":"dewamata",
-            "val":"See you later"
+            "val":"でわまた",
+            "trn":"See you later"
          },
          {
             "key":"mata ashita",
-            "val":"See you Tomorrow"
+            "val":"マタ明日",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: val should be `また明日`

**Comment by tagawa** [comment]
@MathewtheCoder Thank you for these changes. I've added a few comments, but also there's a global edit needed: All the `key` properties should have the English word, and all the `trn` properties should have the pronunciation. This seems to affect all strings except some of the numbers. Would you be able to change them please?

**Comment by MathewtheCoder** [comment]
@tagawa Thanks for pointing it out for me i will change it right away

**Comment by MathewtheCoder** [comment]
@tagawa I have made the corrections you requested

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Japanese Language cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/japanese_language_cheat_sheet_mobile.png?4umVyo7L0N)

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 20]
```diff
@@ -0,0 +1,350 @@
...
+         },
+         {
+            "trn":"nihon",
+            "val":"日本",
+            "key":"Japan"
+         },
+         {
+            "trn":"daigaku",
+            "val":"大学",
+            "key":"college"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please could this be changed to "university"

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 21]
```diff
@@ -0,0 +1,350 @@
...
+         },
+         {
+            "trn":"kaerimasu",
+            "val":"帰ります",
+            "key":"To Return"
+         },
+         {
+            "trn":"taberu",
+            "val":"食べる",
+            "key":"Eat"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please could this be:
trn = `tabemasu`
val = `食べます`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 22]
```diff
@@ -0,0 +1,350 @@
...
+         },
+         {
+            "trn":"jikkō shimasu",
+            "val":"実行します",
+            "key":"To Do"
+         },
+         {
+            "trn":"miru",
+            "val":"見る",
+            "key":"To See"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please could this be:
trn = `mimasu`
val = `見ます`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 23]
```diff
@@ -0,0 +1,350 @@
...
+         },
+         {
+            "trn":"machimasu",
+            "val":"待ちます",
+            "key":"To Wait"
+         },
+         {
+            "trn":"tomaru",
+            "val":"止まる",
+            "key":"To Stop"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please could this be:
trn = `tomarimasu`
val = `止まります`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 24]
```diff
@@ -0,0 +1,350 @@
...
+         },
+         {
+            "trn":"oshieru",
+            "val":"教える",
+            "key":"Teach"
+         },
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please could this be:
trn = `oshiemasu`
val = `教えます`

**Comment by MathewtheCoder** [comment]
@tagawa Yeah i made the corrections.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Japanese Language cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/japanese_language_cheat_sheet_mobile.png?snQIpOVyiS)

**Comment by edgesince84** [comment]
thanks for the corrections @MathewtheCoder 💃 👯

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 25]
```diff
@@ -0,0 +1,350 @@
...
+            "key":"Excuse me/I'm sorry"
+         },
+         {
+            "trn":"domo arigatou gozaimasu",
+            "val":"どうもありがとうございます",
+            "key":"Thank you very much"
+         },
+         {
+            "trn":"itadakimasu",
+            "val":"いただきます。",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please could we remove the period (。) at the end, to make it consistent with other phrases.

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 26]
```diff
@@ -0,0 +1,350 @@
...
+            "val":"五",
+            "key":"5"
+         },
+         {
+            "trn":"roku",
+            "val":"六",
+            "key":"6"
+         },
+         {
+            "trn":"nana",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please change this to `nana/shichi`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 27]
```diff
@@ -0,0 +1,350 @@
+{
+   "id":"japanese_language_cheat_sheet",
+   "name":"Basic Japanese",
+   "description":"A guide to basic Japanese",
+   "metadata":{
+      "sourceName":"Tofugu",
+      "sourceUrl":"https://www.tofugu.com/japanese/important-japanese-words/"
+   },
+   "aliases":[
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: I think we can remove this aliases section.

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 28]
```diff
@@ -0,0 +1,350 @@
...
+            "trn":"itadakimasu",
+            "val":"いただきます。",
+            "key":"Thanks for the food"
+         }
+      ],
+      "Greetings":[
+         {
+            "trn":"Matthew desu",
+            "val":"マシューです",
+            "key":"I'm Mathew"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: We should make this `Matthew` to be the same as the `trn` value (or the other way round).

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 29]
```diff
@@ -0,0 +1,350 @@
...
+         },
+         {
+            "trn":"hajimemashite",
+            "val":"はじめまして",
+            "key":"How do you do"
+         },
+         {
+            "trn":"Anata no namae wa nandesuka",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please change `Anata` to `anata`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 30]
```diff
@@ -0,0 +1,350 @@
...
+            "val":"おやすみなさい",
+            "key":"Good Night"
+         },
+         {
+            "trn":"sayonara",
+            "val":"さようなら",
+            "key":"GoodBye"
+         },
+         {
+            "trn":"Mata aimashō",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please change `Mata` to `mata`

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 31]
```diff
@@ -0,0 +1,350 @@
...
+            "val":"いつ",
+            "key":"When"
+         },
+         {
+            "trn":"nan desu ka",
+            "val":"何ですか",
+            "key":"What is it"
+         },
+         {
+            "trn":"Imananjidesuka",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please change to `Ima nanji desu ka`

**Comment by tagawa** [comment]
Hi @MathewtheCoder. I've made a few more comments which I didn't spot the first time (sorry). Once they've been corrected I'll get a native speaker to double-check, but I think it's looking good.

**Comment by MathewtheCoder** [comment]
Thanks i have made the corrections you requested

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Japanese Language cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/japanese_language_cheat_sheet_mobile.png?YGEA9crVTL)

**Comment by MathewtheCoder** [comment]
@tagawa Is this all the changes i have to make to the file

**Comment by tagawa** [comment]
Thank you @MathewtheCoder. Looks good to me! I'll ask for native speaker help to make sure it's OK.

**Comment by MathewtheCoder** [comment]
@tagawa Thanx man

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 32]
```diff
@@ -0,0 +1,348 @@
...
+            "val":"七",
+            "key":"7"
+         },
+         {
+            "trn":"hachi",
+            "val":"八",
+            "key":"8"
+         },
+         {
+            "trn":"kyu",
```
Comment (Ushinro) [inline share/goodie/cheat_sheets/json/japanese-language.json]: The 'u' should be a long-u sound; two u's or the 'ū' character are typically used to represent this.

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 33]
```diff
@@ -0,0 +1,348 @@
...
+         },
+         {
+            "trn":"kon-nichi-wa",
+            "val":"こんにちは",
+            "key":"Hello/Good Afternoon"
+         },
+         {
+            "trn":"kon-ban-wa",
+            "val":"こんばんは",
+            "key":"Good Afternoon"
```
Comment (Ushinro) [inline share/goodie/cheat_sheets/json/japanese-language.json]: This should say "Good Evening".

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 34]
```diff
@@ -0,0 +1,348 @@
...
+         }
+      ],
+      "Questions":[
+         {
+            "trn":"dare desu ka",
+            "val":"誰ですか",
+            "key":"Who is it"
+         },
+         {
+            "trn":"do desu ka",
```
Comment (Ushinro) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Semantics: The 'o' part is a long-o sound so should either be the 'ō' or have a 'u' at the end to spell "dou".

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 35]
```diff
@@ -0,0 +1,348 @@
...
+            "val":"どこですか",
+            "key":"Where is it"
+         },
+         {
+            "trn":"doushita",
+            "val":"どうした",
+            "key":"What happened"
+         },
+         {
+            "trn":"doshite",
```
Comment (Ushinro) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Semantics: The 'o' part is a long-o sound so should either be the 'ō' or have a 'u' at the end to spell "dou".

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 36]
```diff
@@ -0,0 +1,348 @@
...
+            "val":"こんばんは",
+            "key":"Good Afternoon"
+         },
+         {
+            "trn":"oyasuminasai",
+            "val":"おやすみなさい",
+            "key":"Good Night"
+         },
+         {
+            "trn":"sayonara",
```
Comment (Ushinro) [inline share/goodie/cheat_sheets/json/japanese-language.json]: I have been wrestling with this in my mind as the 'o' is also a long-o sound and technically should either use the 'ō' to spell "sayōnara" or a 'u' after the 'o' to spell "sayounara" but either case looks awkward to me. I would honestly let this one slide but I figured I'd point this out. If I had to choose one, I would use the 'ō' variation.

**Comment by Ushinro** [comment]
Apart from the "Good Evening" mistake, it all looks okay to me. I personally would not bat an eye if the other spelling changes were passed up. I am not sure how extensive you would want this list because there are some exception words that may or may not be worth mentioning. For example, 1000 is "sen" or "issen", 2000 is "nisen", but 3000 is "san_z_en".

**Comment by meteore929** [comment]
- 0 can be pronounced or said 'zero' as well. And it is more common in Japanese to say "zero" than "rei".
- See you later is "ではまた" and not "でわまた".
- Japan can be said "nippon" as well.
- In the verb section, I don't know if you did on purpose, but all the verbs are not in infinitif. If you would like to be in infinitif the following are correct:
To Go
行く
iku

To Return
帰る
kaeru

Eat
食べる
taberu

To Do
やる・実行する
yaru / jikkou suru

To See
見る
miru

To Buy
買う
kau

To Wait
待つ
matu

To Stop
止まる
tomaru

Teach
教える
oshieru

To Speak
話す
hanasu

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 37]
```diff
@@ -294,53 +299,53 @@
...
-            "val":"食べます",
+            "trn":"taberu",
+            "val":"食べる",
             "key":"Eat"
          },
          {
-            "trn":"jikkō shimasu",
-            "val":"実行します",
+            "trn":"yaru/jikkou suru",
+            "val":"やる・実行する",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please could this be changed to just:
"trn":"yaru/suru",
"val":"やる・する",

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 38]
```diff
@@ -294,53 +299,53 @@
...
-            "trn":"kaimasu",
-            "val":"買います",
+            "trn":"kau",
+            "val":"買う",
             "key":"To Buy"
          },
          {
-            "trn":"machimasu",
-            "val":"待ちます",
+            "trn":"matu",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Please change this to "matsu"

**Comment by MathewtheCoder** [comment]
@Ushinro I have made the changes you requested and also as @meteore929 suggested i changed the verb to infintif but have some doubts about some words.Thank you for the suggestions

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 39]
```diff
@@ -215,7 +220,7 @@
             "key":"Who is it"
          },
          {
-            "trn":"do desu ka",
+            "trn":"dou desu ka",
             "val":"どうですか",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Similar to other syntax, this should be "dō desu ka"

File: share/goodie/cheat_sheets/json/japanese-language.json [hunk 40]
```diff
@@ -230,7 +235,7 @@
             "key":"What happened"
          },
          {
-            "trn":"doshite",
+            "trn":"doushite",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/japanese-language.json]: Similar to other syntax, this should be "dōshite"

**Comment by MathewtheCoder** [comment]
@tagawa Is there any other errors associated with verb

**Comment by tagawa** [comment]
Thanks for making the changes so quickly @MathewtheCoder. 

The verbs are acceptable in both forms, e.g. "taberu" and "tabemasu". Personally I prefer "tabemasu" style because that is what visitors to Japan are most likely to hear and use, but the infinitive, e.g. "taberu", is more gramatically correct for a reference, so let's leave it as it is now. We can give it another day or two to see if there is more feedback - if not I'd like to merge this.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Japanese Language cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/japanese_language_cheat_sheet_mobile.png?zWNHQSKupj)

**Comment by MathewtheCoder** [comment]
@tagawa Thanx man

**Comment by tagawa** [comment]
All looks good to me so merging. Thank you again @MathewtheCoder for implementing the changes so many times!

And thanks also to @Ushinro and @meteore929 for the helpful feedback. ありがとうございました！

**Comment by daxtheduck** [comment]
@MathewtheCoder this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**